core/src/main/java/org/mule/runtime/core/execution/ModuleFlowProcessingPhase.java

```diff
@@ -51,6 +51,7 @@
 public class ModuleFlowProcessingPhase
     extends NotificationFiringProcessingPhase<ModuleFlowProcessingPhaseTemplate> {
 
+  public static final String ENABLE_SOURCE_POLICIES_SYSTEM_PROPERTY = "enableSourcePolicies";
```

Comment:
This should be gone in max 2 weeks when @pablolagreca is back. No point putting something temporary in MuleProperties. This is only hear to allow testing with policies in the next two weeks if required.